src/homework10/Part3T30.java

```diff
@@ -0,0 +1,33 @@
+package homework10;
+
+import java.util.*;
+
+//Создать коллекцию, наполнить ее случайными числами. Удалить повторяющиеся числа.
+// Здесь 2 варианта. 1 сделан с ArrayList, 2 с TreeSet
+
+public class Part3T30 {
+    public static void main(String[] args) {
+
+        List<Integer> numbers1 = new ArrayList<>();
+        Set<Integer> numbers2 = new TreeSet<>();
+
+        for (int i = 0; i < 30; i++) {
+            numbers1.add((int) (Math.random() * (-10)) + 10);
+            numbers2.add((int) (Math.random() * (-10)) + 10);
+        }
+
+        System.out.println("ArrayList [before removing] " + numbers1);
+        System.out.println("TreeSet [before removing] " + numbers2);
+
+        for (int i = 0; i < numbers1.size() - 1; i++) {
+            for (int j = 1; j < numbers1.size(); j++) {
+                if (numbers1.get(i).equals(numbers1.get(j))) {
+                    numbers1.remove(j);
+                }
+            }
+        }
+        Collections.sort(numbers1);
+        System.out.println("ArrayList [after removing] " + numbers1);
```

Comment:
ArrayList [before removing] [6, 6, 4, 4, 7, 4, 6, 1, 4, 3, 5, 4, 2, 6, 3, 5, 1, 9, 6, 3, 3, 3, 3, 3, 4, 1, 5, 4, 10, 3]
TreeSet [before removing] [1, 2, 3, 4, 6, 7, 8, 9, 10]
ArrayList [after removing] [1, 3, 3, 4, 5, 6, 10]
TreeSet [after removing] [1, 2, 3, 4, 6, 7, 8, 9, 10]
I got this result, please check ArrayList [after removing]. It contains duplicates. Logic is incorrect, check solution from previous task as tip

Reply:
yeap. my mistake.
fixed

Reply:
please check library